crates/rpc-types-engine/src/payload/mod.rs

```diff
@@ -17,6 +20,8 @@ use error::OpPayloadError;
 #[cfg_attr(feature = "serde", derive(serde::Serialize))]
 #[cfg_attr(feature = "serde", serde(untagged))]
 pub enum OpExecutionPayload {
+    /// V1 payload
+    V1(ExecutionPayloadV1),
```

Comment:
why do we need this again, I thought there is no v1?

Reply:
the disjoint versioning of: new payload method, engine payload AND engine payload body...are seriously confusing